src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -42,6 +43,11 @@ const ResourceLink = (props: IResourceLinkProps) => {
       sampleBody: undefined
     };
     dispatch(setSampleQuery(query));
+    telemetry.trackEvent(eventTypes.LISTITEM_CLICK_EVENT,
+      {
+        ComponentName: componentNames.SET_QUERY_ACTION,
+        sampleUrl
```

Comment:
Property names should be in Pascal case. Should we capture the HTTP Method selected?